Makefile

```diff
@@ -0,0 +1,58 @@
+LIGHTTPD_VERSION = 1.4.77
```

Comment:
hmm, i think this makefile is a bit off. is it intended for local usage (in particular allowing switching between versions etc.), or is it intended to be used within container-build only?

in the latter case, I think it is no necessary, to e.g. take care that build-directories do not clash between versions (also, I think it is somewhat uncommon to use `/tmp` for that). in the first case, it is a better practise to use `mktemp -d` (that way, also not having to handle incremental build-case)